binutils/plan.sh

```diff
@@ -16,7 +17,7 @@ do_prepare() {
 
   # Add explicit linker instructions as the binutils we are using may have its
   # own dynamic linker defaults.
-  dynamic_linker="$(pkg_path_for glibc)/lib/ld-linux-x86-64.so.2"
+  dynamic_linker="$(pkg_path_for core/glibc)/lib/ld-linux-x86-64.so.2"
```

Comment:
The behavior of the `pkg_path_for` helper will accept either a bare package name or qualified origin/name identifier. This is because there can and should only ever be one "glibc" package declared. The dependency locking and sanity checks guarantee this, which is why I was favoring the shorter form.